modules/installation-configuration-parameters.adoc

```diff
@@ -174,3 +192,41 @@ For more information about AWS tags,
 see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources]
 in the AWS documentation.
 |====
+endif::aws[]
+
+ifdef::azure[]
+.Additional Azure parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`machines.platform.azure.type`
+|The Azure VM instance type.
+|VMs that use Windows or Linux as the operating system. See the
+link:https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908[Guest operating systems supported on Azure Stack]
+in the Azure documentation.
+
+`machines.platform.azure.osDisk.diskSizeGB`
+|The Azure disk size for the VM.
+|Integer that represents the size of the disk in GB, for example `512`. The
+minimum supported disk size is `120`.
+
+|`platform.azure.baseDomainResourceGroupName`
+|The name of the resource group that contains the DNS zone for your base domain.
+|String, for example `production_cluster`.
+
+|`platform.azure.region`
+|The name of the Azure region that hosts your cluster.
+|Any valid region name.
+
+|====
+
+[NOTE]
+====
+You cannot customize
```

Comment:
Hello, can you please share more details about this? We have tests configured to use this custom configuration and I have also confirmed I can use the setting with the latest build. 

Example:
```json
compute:
- hyperthreading: Enabled
  name: worker
  platform:
    azure:
      osDisk:
        diskSizeGB: 128
      type: Standard_D4s_v3
      zones:
      - "1"
```

Reply:
Let's see what happens with https://github.com/openshift/installer/pull/2371?

Reply:
This has been merged, but I do not the availability zones in your example.